packages/core/src/Stores/ui-store.js

```diff
@@ -491,6 +492,10 @@ export default class UIStore extends BaseStore {
         this.is_close_uk_account_modal_visible = is_open;
     }
 
+    get is_desktop() {
+        return this.screen_width > MAX_MOBILE_WIDTH;
+    }
+
```

Comment:
Do we really need this flag? As the opposite of `is_mobile` can be desktop.
You can skip this comment as well